extensions/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "msal-node-extensions",
```

Comment:
These files should go in `extensions/msal-node-extensions`, and the name of the package should be `@azure/msal-node-extensions` or similar.

Reply:
Updated